DataFormats/HGCDigi/interface/HGCSample.h

```diff
@@ -71,6 +71,36 @@ class HGCSample {
   bool getToAValid() const { return getWord(kToAValidMask, kToAValidShift); }
   uint32_t operator()() { return value_; }
 
+  /**
+     @short Data Model Evolution
+  */
+  static uint32_t convertToNewFormat(uint32_t valueOldForm, bool toaFiredOldForm) {
```

Comment:
shouldn't this have a version in the name?
"New" will become obsolete after another update.

Reply:
addressed.